docs/astro.config.mjs

```diff
@@ -233,6 +233,10 @@ export default defineConfig({
             {
               label: 'Make recurring payments',
               link: '/guides/make-recurring-payments/'
+            },
+            {
+              label: 'Split payments',
+              link: '/guides/split-payments/'
```

Comment:
I'm not sure what to suggest yet, but we should probably have a more descriptive title. We can discuss later.